docs/courses/robotics/3.1-pumps-and-pipettes.md

```diff
@@ -177,8 +230,29 @@ tic.deenergize()
 tic.enter_safe_start()
 ```
 
+#### Unexpected Applications 🎭
+
+- 🎰 Slot machines use them for spinning reels
+- 🎹 Electronic musical instruments for key action
+- 🚗 Some car headlights use them for adaptive lighting systems
+
 ### Fan Control
 
+Temperature control is crucial in many lab setups! 🌡️ Fan controllers help in:
+
+- 🧊 Maintaining precise temperatures in incubators
```

Comment:
Let's also mention that many custom lab setups use magnets attached to fans to control stir bars for chemistry experiments, e.g., [pioreactor](https://pioreactor.com/).